src/databricks/labs/ucx/source_code/python_ast_util.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import ast
+from dataclasses import dataclass
+
+
+@dataclass
+class Span:
+    """Represents a (possibly multiline) source code span."""
+
+    start_line: int
+    start_col: int
+    end_line: int
+    end_col: int
+
+
+class AstUtil:
```

Comment:
can we move these methods to `ASTLinter`?.. 

this whole file largely overlaps with `python_linter.py`

Reply:
It's possible but very hard to tell, ASTLinter is undocumented and rather complex.
I'm also a bit worried about the over-use of class inheritance - we could easily do composition or simple static calls for utilities such as these.

Reply:
I moved everything over under ASTLinter in the new revision.